pages/test_analytics/ruby_collectors.md

```diff
@@ -63,6 +63,14 @@ Failure/Error: allow_any_instance_of(Object).to receive(:sleep)
 
 You can fix them by being more specific in your stubbing by replacing `allow_any_instance_of(Object).to receive(:sleep)` with `allow_any_instance_of(TheClassUnderTest).to receive(:sleep)`.
 
+### Troubleshooting test grouping issues
+
+In RSpec anonymous test cases are supported, tests are automatically named based on the subject and/or inputs to the expectations within the test. However, this can lead to unstable test names across different test runs, incorporating elements such as object IDs, database IDs, timestamps, and more.
```

Comment:
Maybe tweak this wording a little to:

```suggestion
RSpec supports anonymous test cases—tests are automatically named based on the subject and/or inputs to the expectations within the test. However, this can lead to unstable test names across different test runs, incorporating elements such as object IDs, database IDs, timestamps, and more.
```